Extension/src/nativeStrings.json

```diff
@@ -178,15 +178,38 @@
         "hint": "Refers to C++ template specializations. This is part of a description indicating there are N additional specializations of a function.  %d is replaced with that number."
     },
     "expands_to": "Expands to:",
+    "attention_label": "Attention:",
```

Comment:
@Colengms Should these be added to the end so it doesn't trigger re-translations for all the strings after this?